libexec/goenv-install

```diff
@@ -88,8 +89,9 @@ if [ "$rtn" == "1" ]
 then
     download="http://golang.org/dl/go${version}.${platform}-${arch}${extra}.tar.gz"
 else
-    download="https://go.googlecode.com/files/go${version}.${platform}-${arch}${extra}.tar.gz"
+    download="https://storage.googleapis.com/golang/go${version}.${platform}-${arch}${extra}.tar.gz"
```

Comment:
It looks like this is only changing the URL for downloads up to 1.2. Should this be a change to the first download url (golang.org), instead?